lib/fictium/exporters/open_api/v3_exporter/content_formatter.rb

```diff
@@ -2,18 +2,26 @@ module Fictium
   module OpenApi
     class V3Exporter
       class ContentFormatter
+        APP_JSON = 'application/json'.freeze
+
         def format(http_object, default = nil)
           type = (http_object.presence && http_object[:content_type].presence) || default
           return if type.blank?
 
           {}.tap do |content|
             media_type = {
-              example: http_object[:body]
+              example: body_formatter(http_object[:content_type], http_object[:body])
             }
             media_type[:schema] = http_object[:schema] if http_object[:schema].present?
             content[type.to_sym] = media_type
           end
         end
+
+        protected
+
+        def body_formatter(content_type, body)
```

Comment:
as a suggestion, while not required, because this is a method and not a property, it can be better written as such:
`format_body(content_type, body)` not a serious issue thou.

Reply:
it's ok, I take your comments